### ISSUE TITLE
TypeError: Cannot read property 'staticAlloc' of undefined

### ISSUE THREAD

**Issue by thecooltechguy** [ucsd-cse231-w21/chocopy-wasm-compiler#82]
Hi,

Ii we run the following program:

```
i : int = 0
```

we get the following error:

```
TypeError: Cannot read property 'staticAlloc' of undefined
```

Specifically, if you enter this in the web UI, hit "run", and then hit "run" again, you will see this error.
You can also reproduce this error by hitting "run", "reset", and then "run" again.

It will only go away if we refresh the page after encountering the error.

Does anyone know what could be causing this?

**Comment by R167** [comment]
Do you have a stack trace from web inspector? Sounds like GC?

**Comment by thecooltechguy** [comment]
Sorry forgot to add that to the original post. I've added it now.

**Comment by suhas-goutham** [comment]
Yeah we are facing the same issue when we try printing multiple strings like this :- 
```
print("ABC")
print("XYZ")
```

**Comment by suhas-goutham** [comment]
I think this issue gets solved if you refresh the html page. Atleast that's what worked for us.
